{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--f01ed2e1-ec3a-43ed-b1e7-43a6a1c70d1a",
    "created": "2020-05-26T20:29:44.069304Z",
    "modified": "2020-05-26T20:29:44.069304Z",
    "name": "CVE-2018-10879",
    "description": "A flaw was found in the Linux kernel's ext4 filesystem. A local user can cause a use-after-free in ext4_xattr_set_entry function and a denial of service or unspecified other impact may occur by renaming a file in a crafted ext4 filesystem image."
}